src/hyperloglog.rs

```diff
@@ -551,4 +562,19 @@ mod tests {
         assert_eq!(hll.count(), 571);
         assert!(!hll.is_empty());
     }
+
+    #[test]
+    fn slice() {
+        // `HyperLogLog<T>` with a dynamically-sized `T`.
+        let mut h: HyperLogLog<[u8]> = HyperLogLog::new(4);
+        {
+            // Can `add` and `extend` from vecs or arrays.
+            let v1 = vec![0; 64];
+            h.add(&v1);
+            let v2 = vec![&v1[..]];
+            h.extend(v2);
```

Comment:
can you please pull out the extend part into a separate test?

Reply:
Done.